odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -52,8 +52,12 @@ def _fbp_filter(norm_freq, filter_type, frequency_scaling):
     ----------
     norm_freq : `array-like`
         Frequencies normalized to lie in the interval [0, 1].
-    filter_type : {'Ram-Lak', 'Shepp-Logan', 'Cosine', 'Hamming', 'Hann'}
+    filter_type : {'Ram-Lak', 'Shepp-Logan', 'Cosine', 'Hamming', 'Hann',
```

Comment:
Why don't we just use the name `filter`?

Reply:
Don't wanna introduce a breaking change